queries/buildings.jinja2

```diff
@@ -1,13 +1,20 @@
 SELECT
-    name,
+    -- drop name when the item is a POI - in that case the name
+    -- probably refers to the POI. instead, we can use the house
+    -- name tag, if it has been set. likewise for landuse areas
+    -- where we might want to label the landuse rather than the
+    -- building.
+    CASE WHEN mz_poi_min_zoom IS NULL AND
+              (mz_is_landuse IS NULL OR mz_is_landuse = FALSE)
+         THEN name
+         ELSE "addr:housename"
+    END AS name,
     way_area::bigint AS area,
     osm_id AS __id__,
-    "building:part",
+    'openstreetmap.org' AS source,
```

Comment:
For buildings the source will always be 'openstreetmap.org`, no? Is this for consistency with other layers? Or would a post process step create features that have a different source or none at all?